app/services/workflow_errors_reporter.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+# This class will send a put ReST call to the preservationWF via faraday. This also parses
+# out a shorter error message, that only includes the moab validation error.
+class WorkflowErrorsReporter
+
+  def self.update_workflow(druid, error_message)
+    error_message.select { |error| request(druid, 'moab-valid', moab_error(error)) if error.key?(PreservedObjectHandlerResults::INVALID_MOAB) }
+  end
+
+  private_class_method def self.conn
+    if Settings.workflow_services.url.present?
+      @connection ||= Faraday.new(url: Settings.workflow_services.url)
+      Rails.logger.log(warn, 'no workflow hookup - assume you are in test or dev environment')
+    end
+    @connection
+  end
+
+  private_class_method def self.request(druid, process_name, error_message)
+    return unless conn
+    @connection.put do |request|
+      request.headers['content-type'] = "application/xml"
+      request.url  "/workflow/dor/objects/druid:#{druid}/workflows/preservationWF/#{process_name}"
+      request.body = "<process name='#{process_name}' status='error' errorMessage='#{error_message}'/>"
+    end
+  end
+
+  private_class_method def self.moab_error(error)
+    /^.*\)(.*)$/.match(error[PreservedObjectHandlerResults::INVALID_MOAB]).captures.first
```

Comment:
would it be clearer / appropriate to just split on `) ` and take the last piece?  Also, note the space after the paren.

And would that be generalized, so you could use it for any errors you wanted, not just moab errors?